.github/workflows/windows.yml

```diff
@@ -82,9 +82,9 @@ jobs:
     - name: Install libjpeg (vcpkg)
       if: steps.cache-vcpkg.outputs.cache-hit != 'true'
       run: .\vcpkg\vcpkg install libjpeg-turbo:x64-windows
-    - name: Install lua (vcpkg)
+    - name: Install luajit (vcpkg)
       if: steps.cache-vcpkg.outputs.cache-hit != 'true'
-      run: .\vcpkg\vcpkg install lua:x64-windows
+      run: .\vcpkg\vcpkg install luajit:x64-windows
```

Comment:
Why not both? At native-windows we have both with meson wraps

Reply:
"Why not both" is a great question.

Reply:
Done.